cmd/feed-ingress/main.go

```diff
@@ -168,6 +174,9 @@ func init() {
 		"Comma separated list of CIDRs to trust when determining the client's real IP from "+
 			"frontends. The client IP is used for allowing or denying ingress access. "+
 			"This will typically be the ELB subnet.")
+	flag.StringVar(&nginxDefaultSslPath, "default-ssl-path", defaultNginxDefaultSslPath,
```

Comment:
shouldn't this arg just be ssl-path, and have a default value, instead of being called default-ssl-path?